applications/services/cli/cli_vcp.c

```diff
@@ -190,6 +221,10 @@ static int32_t vcp_worker(void* context) {
         }
 
         if(flags & VcpEvtStop) {
+            if(vcp->connected) {
```

Comment:
switching between cli and any other tab in lab is not working as intended

Reply:
This bug has existed prior to this PR. As discussed outside of GitHub, we'll fix this later [FL-3913].